napalm/eos/utils/versions.py

```diff
@@ -0,0 +1,70 @@
+"""Some functions to work with EOS version numbers"""
+import re
+
+
+class EOSVersion:
```

Comment:
I like this, and I have the feeling we could introduce this sort of classes to any driver (I sort of doubt we can find something generic that works for every platform). That'd be very helpful in many contexts - either NAPALM internal, or users in their environment.